contracts/eden/include/migrations.hpp

```diff
@@ -14,19 +16,42 @@ namespace eden
       uint32_t migrate_some(eosio::name contract, uint32_t max_steps) { return max_steps; }
    };
    EOSIO_REFLECT(no_migration<0>);
-   using migration_variant = std::variant<migrate_auction_v0, no_migration<0>>;
+   using migration_variant = std::variant<migrate_auction_v0, migrate_account_v0, no_migration<0>>;
```

Comment:
If the contract before this PR is currently at index 1, then it looks like deserialization will malfunction when this PR is installed. Did you mean this instead?

```
using migration_variant = std::variant<migrate_auction_v0, no_migration<0>, migrate_account_v0, no_migration<1>>;
```

Reply:
I intended to do that iff we merge to production before this PR is merged.